ensemble/ctf/window_function_component.py

```diff
@@ -39,14 +65,39 @@ def values(self):
 class WindowOpacityNode(OpacityNode):
     """ An `OpacityNode` with a non-zero radius and a trapezoidal shape.
     """
+    # The function used to generate the opacity curve.
+    window_func = Property(Callable, depends_on=['window_type'])
+
+    # The name of the window function
+    window_type = Enum(DEFAULT_WINDOW_TYPE, values='_window_types')
+    _window_types = List(WINDOW_FUNCTIONS.keys())
```

Comment:
You could do 

`WindowEnum = Enum(WINDOW_FUNCTIONS.keys())` 

and then 
`window_type = WindowEnum(DEFAULT_WINDOW_TYPE)`

Traits also lets you do 

`window_type = Enum(DEFAULT_WINDOW_TYPE, *WINDOW_FUNCTIONS.keys())`

which feels wrong because you're effectively providing `DEFAULT_WINDOW_TYPE` twice